test/src/unit-enumerations.cc

```diff
@@ -1526,8 +1540,9 @@ TEST_CASE_METHOD(
   auto array = get_array(QueryType::READ);
   array->load_all_enumerations();
 
-  auto schema = make_shared<ArraySchema>(HERE(), array->array_schema_latest());
-  auto enmr1 = schema->get_enumeration("test_enmr");
+  const auto& schema = array->array_schema_latest();
+  auto schema_ptr = schema.clone();
```

Comment:
Same as above. The one liner means you don't have to change the following lines from `schema` to `schema_ptr`.